ceph-deploy-pull-requests/setup/playbooks/tasks/ubuntu.yml

```diff
@@ -3,7 +3,7 @@
     action: apt update_cache=yes
 
   - name: install python requirements
-    action: apt pkg={{ item }} state=installed
+    action: apt pkg={{ item }} state=present
```

Comment:
Do we need to define the parameter at all? [The default is actually `present`](https://docs.ansible.com/ansible/latest/modules/apt_module.html#parameter-state).

Reply:
@rzarzynski thanks! updated and repushed.